### ISSUE TITLE
There's no way to download zip anymore I can't download

### ISSUE THREAD

**Issue by James-the-Coding-Gamer** [Sycnex/Windows10Debloater#276]
So I'm trying to download the zip file and it's only giving me the option to view code and go to file and then I click on it and it's just the code

**Comment by EddyLuten** [comment]
That's right, the code *is* the program you want to run (Code -> Download ZIP). Download the ZIP, extract it, and simply follow the instructions in the README.md file.

**Comment by alanngo** [comment]
Run this command
`$ git clone https://github.com/Sycnex/Windows10Debloater.git`
